### PR TITLE
fix: remove blockWeightsStore

### PR DESCRIPTION
**Pull request by TarikGul**
Part 2 via https://github.com/paritytech/substrate-api-sidecar/pull/890

This will remove the blockWeightsStore as it is not needed anymore. `api.at` does all the decoration for us so we can retrieve the weights via the `historicApi.consts` every time when applicable. 

Todo's:
- [x] Update all the docs

### CODE REVIEW

File: src/services/blocks/BlocksService.ts [hunk 1]
```diff
@@ -537,7 +534,7 @@ export class BlocksService extends AbstractService {
 
 		// Now that we know the exact runtime supports fee calcs, make sure we have
 		// the weights in the store
```
Comment (jsdw) [inline src/services/blocks/BlocksService.ts:536]: Nit: I think this comment is now out of date :)

**Review by jsdw — APPROVED** [review]
Loving how much code was removed, and it feels simpler now :)

**Review by 0xslipk — APPROVED** [review]
LGTM

File: src/services/blocks/BlocksService.spec.ts [hunk 2]
```diff
@@ -309,23 +281,43 @@ describe('BlocksService', () => {
...
 			cache.reset();
 
-			const historicApi = createApiWithAugmentations(
-				polkadotMetadataV29.toHex()
-			) as ApiDecoration<'promise'>;
+			/**
+			 * This is the mockApi adjusted to mock a runtime that uses
+			 * consts.system.blockWeights for its weight nomination.
+			 */
+			const mockHistoricApiAdjusted = {
```
Comment (emostov) [inline src/services/blocks/BlocksService.spec.ts:288]: any reason for the term "Adjusted"?

**Review by emostov — APPROVED** [review]
nice optimization!

File: src/services/blocks/BlocksService.ts (hunk 1 above)
Comment (TarikGul) [reply src/services/blocks/BlocksService.ts:536]: Yea will get removed/updated. Good catch!

File: src/services/blocks/BlocksService.spec.ts (hunk 2 above)
Comment (TarikGul) [reply src/services/blocks/BlocksService.spec.ts:288]: No particular reason, any suggestions are definitely welcome. I found the naming a little awkward so i stuck with adjusted

**Review by niklasad1 — APPROVED** [review]

**Pull request merged**